identity/aziot-identityd/src/lib.rs

```diff
@@ -379,14 +379,24 @@ impl auth::authorization::Authorizer for SettingsAuthorizer
 			crate::auth::OperationType::GetModule(m) => {
 				if let crate::auth::AuthId::LocalPrincipal(creds) = o.auth_id {
 					if let Some(p) = self.pmap.get(&crate::auth::Uid(creds.0)) {
-						return Ok(p.name.0 == m && p.id_type == Some(aziot_identity_common::IdType::Module))
+						let allow_id_type = match &p.id_type {
+							Some(i) => i.contains(&aziot_identity_common::IdType::Module),
```

Comment:
We would also allow device type host process to query module identity. 
Does this line mean that device type can't?

Reply:
No, this just requires the principal to specify idtype = module in order to authorize.

Reply:
I see. So for a host process to access both device and module identity, it needs to specify idtype = ["device", "module"]. Got you.